pkg/sql/sem/builtins/geo_builtins.go

```diff
@@ -4980,6 +4980,26 @@ See http://developers.google.com/maps/documentation/utilities/polylinealgorithm`
 			Volatility: tree.VolatilityImmutable,
 		},
 	),
+	"st_unaryunion": makeBuiltin(
+		defProps(),
+		geometryOverload1(
+			func(_ *tree.EvalContext, g *tree.DGeometry) (tree.Datum, error) {
+				res, err := geomfn.UnaryUnion(g.Geometry)
+				if err != nil {
+					return nil, err
+				}
+				return tree.NewDGeometry(res), nil
+			},
+			types.Geometry,
+			infoBuilder{
+				info: `Similar to ST_Union, but does dissolve boundaries between components of a multipolygon (invalid) and does perform union between the components of a geometrycollection.
```

Comment:
I think this explanation is enough and more to the style of our comments:
```
Returns a union of all geometries, but does not dissolve boundaries of a MultiPolygon.
```

Reply:
> Unlike ST_Union, ST_UnaryUnion does dissolve boundaries between components of a multipolygon (invalid)

It actually does dissolve boundaries :)